lib/compat/wordpress-6.5/fonts/class-wp-font-utils.php

```diff
@@ -20,11 +20,41 @@
 	 * @access private
 	 */
 	class WP_Font_Utils {
+
+		/**
+		 * Adds surrounding quotes to font family names that contain special characters.
+		 *
+		 * It follows the recommendations from the CSS Fonts Module Level 4.
+		 * @link https://www.w3.org/TR/css-fonts-4/#font-family-prop
+		 *
+		 * @since 6.5.0
+		 * @access private
+		 *
+		 * @see sanitize_font_family()
+		 *
+		 * @param string $item A font family name.
+		 * @return string The font family name with surrounding quotes if necessary.
+		 */
+		private static function maybe_add_quotes( $item ) {
+			// Match any non alphabetic characters (a-zA-Z), dashes -, or parenthesis ().
+			$regex = '/[^a-zA-Z\-()]+/';
+			$item  = trim( $item );
+			if ( preg_match( $regex, $item ) ) {
+				// Removes leading and trailing quotes.
+				$item = preg_replace( '/^["\']|["\']$/', '', $item );
```

Comment:
Using `trim` and specifying quotes to strip here is probably slightly more performant than using a regex.